boards/common/stm32/include/cfg_usbdev_fs.h

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
For the moment, since there's only one board affected, I would move this config to the p-nucleo-wb55 board itself. Because we are not sure this configuration can be reused as-is with other ST boards (especially the nucleos).